_data/featured-apps.yml

```diff
@@ -2,12 +2,12 @@
   description: Visualize where the money comes from and how it's spent.
   image: openbudget.png
   url: http://openbudgetsac.org/
-- name: SacSOS
-  description: Find community services in Sacramento County.
-  image: sacsos.png
-  url: http://sacsos.org/
-- name: Polling Place Accessibility
-  description: Learn about the accessibility of polling locations.
-  image: pollingplaces.png
-  url: http://abustamam.github.io/polling-place-accessibility/
+- name: Trash-AI
+  description: Developing machine learning technology to identify trash you find on the street.
```

Comment:
Developing machine learning technology to identify trash in images.

Reply:
lol yes that makes much more sense, thanks